keras_cv/utils/preset_utils_test.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+
+import numpy as np
+import pytest
+from absl.testing import parameterized
+
+from keras_cv.models import DeepLabV3Plus
+from keras_cv.models import ImageClassifier
+from keras_cv.models import RetinaNet
+from keras_cv.models import YOLOV8Detector
+from keras_cv.tests.test_case import TestCase
+from keras_cv.utils import preset_utils
+
+
+class PresetUtilsTest(TestCase):
+    @parameterized.parameters(
+        (ImageClassifier, "resnet50_v2_imagenet_classifier", "classification"),
```

Comment:
How'd you come up with this list? Are all these model testing something substantially different?

Reply:
This list was chosen for simple models across each of the task types (classification, object detection, and segmentation).